9.1-2.2/initdb-postgis.sh

```diff
@@ -6,15 +6,15 @@ set -e
 export PGUSER="$POSTGRES_USER"
 
 # Create the 'template_postgis' template db
-psql --dbname="$POSTGRES_DB" <<- 'EOSQL'
+"${psql[@]}" <<- 'EOSQL'
 CREATE DATABASE template_postgis;
 UPDATE pg_database SET datistemplate = TRUE WHERE datname = 'template_postgis';
 EOSQL
 
 # Load PostGIS into both template_database and $POSTGRES_DB
 for DB in template_postgis "$POSTGRES_DB"; do
 	echo "Loading PostGIS extensions into $DB"
-	psql --dbname="$DB" <<-'EOSQL'
+	"${psql[@]}" --dbname="$DB" <<-'EOSQL'
```

Comment:
Based on the passing test, I'm assuming that `psql` is cool receiving `--dbname=` twice and just uses the second one?